src/query/procedure/py_module.cpp

```diff
@@ -2052,6 +2052,86 @@ PyObject *PyPathMakeWithStart(PyTypeObject *type, PyObject *vertex) {
   return py_path;
 }
 
+// clang-format off
+struct PyLogger {
+  PyObject_HEAD
+};
+// clang-format on
+
+void PyLoggerAddLog(const enum mgp_log_level level, const char *out) { mgp_log(level, out); }
```

Comment:
I think here the return value of `mgp_log` is ignored, so it should result in a compilation error in my opinion.

Reply:
I changed mgp_log to be void as it doesn't return value, so all good here now I think